latch_cli/services/local_dev.py

```diff
@@ -33,10 +38,41 @@
 EXEC_COMMANDS = ["shell", "run", "run-script"]
 LIST_COMMANDS = ["ls", "list-tasks"]
 
+COMMAND_EXPRESSION = re.compile("^(run\s.+|run-script\s.+|shell|list-tasks|ls.*)$")
```

Comment:
you should actually be able to include arbitrary whitespace characters before your first command and still get desired behavior (try with bash)